### DISCUSSION TITLE
Use of theme problem padding

### DISCUSSION THREAD

**Discussion by BITRU** [unocss/unocss#1013] [Q&A]
I am trying to move from taillwind, but got problem in figuring out theme. 
```
theme: {
    breakpoints: {
      'sm': '576px',
      'md': '768px',
      'lg': '992px',
      'xl': '1200px',
      '2xl': '1400px',
    },
    container: {
      center: true,
      padding: {
        'DEFAULT': '1rem',
        'sm': '2rem',
        'lg': '4rem',
        'xl': '5rem',
        '2xl': '9rem',
      },
    },
  },
```
for some reason part with breakpoints work fine. But class='container' does not get padding or get centred. 
and also how can I extend theme with classes. like add bg-gray-200 to container.

**Comment by chu121su12** [comment]
The theme object in unocss is not the same as it might be used in tailwind. One way to add extra properties in a class is to use [shortcut](https://github.com/unocss/unocss#shortcuts) to then use `container-x` and `bg-x`.

**Reply by BITRU** [reply]
but how I can pre-configure used classes and tags like h1 if i want override default size? I have to do extra work adding like class='H1-x' ?

**Reply by chu121su12** [reply]
You can still add/override theme object [[Theme](https://github.com/unocss/unocss#extend-theme)]. 
Also I believe overriding default tag style is best done in preflight.

**Reply by BITRU** [reply]
Now I am totally confused.  So why my container object did not get padding override the way I did I first post.?

**Reply by chu121su12** [reply]
There is no feature in unocss to override css from theme object (yet? Please open issue/PR/discussion to discuss this feature).
The theme object used mostly for obtaining property value(s).
